ckan/Dockerfile.dev

```diff
@@ -56,4 +56,6 @@ RUN for d in $APP_DIR/patches/*; do \
                 cd $SRC_DIR/`basename "$d"` && echo "$0: Applying patch $f to $SRC_DIR/`basename $d`"; patch -p1 < "$f" ; \
             done ; \
         fi ; \
-    done
\ No newline at end of file
+    done
+
+COPY setup/start_ckan_development.sh ${APP_DIR}
```

Comment:
If I read the original Dockerfile it feels like the intend was to install any extensions already earlier in this script (see: https://github.com/CivityNL/ckan-docker/blob/master/ckan/Dockerfile.dev#L15)